extensions/amp-bind/0.1/bind-expr-impl.jison

```diff
@@ -1,41 +1,45 @@
 /** shared vars and functions */
 
 %{
+// Shortcuts for common functions.
+var toString = Object.prototype.toString;
+var hasOwnProperty = Object.prototype.hasOwnProperty;
+
 // For security reasons, must not contain functions that mutate the caller.
 var functionWhitelist =
 {
   '[object Array]':
-    [
-      Array.prototype.concat,
-      Array.prototype.includes,
-      Array.prototype.indexOf,
-      Array.prototype.join,
-      Array.prototype.lastIndexOf,
-      Array.prototype.slice,
-    ],
+    {
```

Comment:
Good for us we have Justin on our side :) Lets also test them.

`types.js#map` can help with creating such an object from a literal.